blazor/common/custom-resource-generator.md

```diff
@@ -38,7 +38,10 @@ Refer to the following steps to download the custom resources in CRG:
 3. Now, the export request has been initiated for the selected component(s). You will receive an email with a download link within a minute( Depending on the selected component counts and bundle type minified or unminified ).
 ![Blazor Bundle custom resources](images/mail-with-download-link.png)
 4. The final output contains the custom interop script and styles for the selected components and an **import.json** file, which stores the current settings.
-![Final output of customized resources in Blazor](images/customized-resources.png)
+
+   ![Final output of customized resources in Blazor](images/customized-resources.png)
+
+N> If you download resources of Material and Tailwind themes, either alone or with other themes, the 'styles' folder will contain CSS files along with a 'customized' folder. The CSS files within the 'customized' folder do not include online Google font dependencies of the Material and Tailwind themes. For more details, please refer to [this](https://blazor.syncfusion.com/documentation/appearance/themes#render-syncfusion-components-in-offline-with-material-and-tailwind-themes).
```

Comment:
![image](https://user-images.githubusercontent.com/33709971/228461380-81f0731a-5cc1-4e66-951f-16b669729b2b.png)
use this ChatGPt generated content.

`"When you download the resources of Material and Tailwind themes, either separately or with other themes, you will find a 'styles' folder that contains CSS files along with a 'customized' folder. The CSS files within the 'customized' folder do not include the online Google font dependencies associated with the Material and Tailwind themes. For more details, please refer to [this](https://blazor.syncfusion.com/documentation/appearance/themes#render-syncfusion-components-in-offline-with-material-and-tailwind-themes) resource."`

Reply:
The provided content has been used - c03ebf902d132ecf72a194739c34ecff6e80b9f0